Content.Client/GameObjects/Components/HUD/Inventory/InventoryInterfaceController.cs

```diff
@@ -30,23 +30,11 @@ public virtual void Initialize()
         public virtual void PlayerAttached()
         {
             GameHud.InventoryButtonVisible = true;
-            GameHud.InventoryButtonToggled = b =>
```

Comment:
Shouldn't this class be nuked entirely or is it out of scope? e.g. stuff like the Window property definitely shouldn't exist anymore.

Reply:
Definitely out of scope. A lot of it is entangled with the HUD inventory code and rewriting a bunch of inventory code in a PR to rearrange the HUD is definitely not something I am going to do.